kafka-plugins-0.10/src/main/java/io/cdap/plugin/connector/KafkaConnector.java

```diff
@@ -57,8 +57,8 @@
  */
 @Plugin(type = Connector.PLUGIN_TYPE)
 @Name(KafkaConnector.NAME)
-@Category("Messaging System")
-@Description("Kafka Connector to browse and sample topics")
+@Category("Messaging Systems")
+@Description("Kafka Connector to browse and sample topics for kafka 0.10 and above")
```

Comment:
what versions of Kafka have we tested with? The current version is 2.8.0, I would be surprised if it works with everything above 0.10.

Reply:
Only tested with 0.10.2, I will change this description